Modules/SubModule_Import/xASL_imp_NII2BIDS_Session.m

```diff
@@ -37,11 +37,10 @@ function xASL_imp_NII2BIDS_Session(imPar, bidsPar, studyPar, listSessions, nameS
         else
             sessionLabel = ['ses-' listSessions{iSession}(5:end)];
         end
-
-        if ~exist(fullfile(imPar.BidsRoot,['sub-' subjectLabel],sessionLabel),'dir')
-            xASL_adm_CreateDir(fullfile(imPar.BidsRoot,['sub-' subjectLabel],sessionLabel));
-            xASL_adm_CreateDir(fullfile(imPar.BidsRoot,['sub-' subjectLabel],sessionLabel,'perf'));
-        end
+        
+        xASL_adm_CreateDir(fullfile(imPar.BidsRoot,['sub-' subjectLabel],sessionLabel));
```

Comment:
Yes so you can even remove this folder, it is a folder layer above the next folder that you create, so `xASL_adm_CreateDir` will create it automatically!

Reply:
Yeah, but we did it like this before and I didn't want to change even more of Jans code. You always insist on me making smaller but more gradual changes and now we are starting to optimize each line of code again...

Reply:
That's true :) But this line is not used, as the next line does exactly the same. I would remove it.

Reply:
Good to make minimal changes only. But agree to remove the first line here as not needed anymore. I did that already in a recent commit